<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudienceInvitePKWidget</class>
 <widget class="QWidget" name="AudienceInvitePKWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YY斗歌</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="JoinBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>89</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
border-image: url(:/image/AudienceInvitePKWidget/bt_want_hover.png);}
QPushButton{border-image: url(:/image/AudienceInvitePKWidget/bt_want.png);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="PropListWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>118</y>
     <width>240</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>93</y>
     <width>210</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 12px &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>想看主播表演以下节目么？还等什么！</string>
   </property>
  </widget>
  <widget class="QLabel" name="InviteNumberInfoLabel">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>212</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 12px &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="InviteSongInfoLabel">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>54</y>
     <width>260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(232, 143, 206);
font: 12px &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>1</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/image/AudienceInvitePKWidget/bt_close.png);}
QPushButton:hover{border-image: url(:/image/AudienceInvitePKWidget/bt_close_hover.png);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
